sled-agent/src/rack_setup/service.rs

```diff
@@ -2,23 +2,87 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-//! Rack Setup Service implementation
-
-use super::config::{HardcodedSledRequest, SetupServiceConfig as Config};
-use crate::bootstrap::config::BOOTSTRAP_AGENT_PORT;
+//! Rack Setup Service (RSS) implementation
```

Comment:
Thank you for this comment!  (I wonder if it belongs in sled-agent/src/rack_setup/lib.rs so it'd be more easily found?)

Reply:
Do you mean `sled-agent/src/rack_setup/mod.rs` ? I'll add doc comments there to help discovery (a6593901500f442723a0d2f82d60d667c4cab425), but I think this comment is tightly coupled with this file, as the body of `service.rs` is the implementation described here.